### PR TITLE
`<LazyFrame>$fetch()` 

### LINKED ISSUE #283: Implement `$fetch()`
= `collect`ing just a few rows of data

https://pola-rs.github.io/polars/py-polars/html/reference/lazyframe/api/polars.LazyFrame.fetch.html#polars.LazyFrame.fetch

### PR DESCRIPTION
**Pull request by sorhawell**
Close #283

### CODE REVIEW

**Comment by sorhawell** [comment]
tests and better docs missing

File: R/dataframe__frame.R [hunk 1]
```diff
@@ -1527,3 +1527,13 @@ DataFrame_glimpse = function(..., return_as_string = FALSE) {
...
+
+
+#' @title Fetch limited number of rows of DataFrame
+#' @keywords DataFrame
+#' @inherit LazyFrame_fetch
+#' @examples
+#' pl$DataFrame(iris)$fetch(2)
+DataFrame_fetch = function(n_rows) {
+  self$lazy()$fetch(n_rows)
+}
```
[the remark this answers is not in the document]
Comment (sorhawell) [reply R/dataframe__frame.R:1539]: sry I was in auto-mode

[the remark this answers is not in the document]
Comment (etiennebacher) [reply R/dataframe__frame.R:1539]: no worries ^^

File: R/lazyframe__lazy.R [hunk 2]
```diff
@@ -1183,13 +1188,100 @@ LazyFrame_dtypes = method_as_property(function() {
...
+#' run on minimal required memory.
+#' @param predicate_pushdown  Boolean. Applies filters as early as possible / at
+#' scan level.
+#' @param projection_pushdown  Boolean. Applies filters as early as possible / at
+#' scan level.
+#' @param simplify_expression  Boolean. Cache subtrees/file scans that are used
+#' by multiple subtrees in the query plan.
+#' @param slice_pushdown  Boolean. Only load the required slice from the scan
+#' level. Don't materialize sliced outputs (e.g. `join$head(10)`).
+#' @param common_subplan_elimination  Boolean. Cache subtrees/file scans that
```
Comment (etiennebacher) [inline R/lazyframe__lazy.R:1213]: This should be updated to use `comm_subplan_elim` and `comm_subexpr_elim` as in `collect()`

**Review by etiennebacher — COMMENTED** [review]
@sorhawell the only thing needed is to update the parameters to use `comm_subplan_elim` and `comm_subexpr_elim`. I can do it tomorrow if you want, but if anyone can do it before feel free to do so

**Pull request merged**